azure-kusto-ingest/azure/kusto/ingest/resource_manager.py

```diff
@@ -0,0 +1,99 @@
+"""
+    This module is serve as a cache to all resources needed by the kusto ingest client
+"""
+
+from azure.kusto.data import KustoClient
+from .connection_string import _ConnectionString
+
+from datetime import datetime
+from datetime import timedelta
+import random
+
+class IngestClientResources:
```

Comment:
Is that class public on purpose?

Reply:
Changed to privet (meaning to "_IngestClientResources")